src/SqlConverters.cs

```diff
@@ -68,16 +70,20 @@ internal class SqlGenericsConverter<T> : IAsyncConverter<SqlAttribute, IEnumerab
         {
             private readonly IConfiguration _configuration;
 
+            private readonly ILogger _logger;
+
             /// <summary>
             /// Initializes a new instance of the <see cref="SqlGenericsConverter<typeparamref name="T"/>"/> class.
             /// </summary>
             /// <param name="configuration"></param>
+            /// <param name="loggerFactory">Logger Factory for creating an ILogger</param>
             /// <exception cref="ArgumentNullException">
             /// Thrown if the configuration is null
             /// </exception>
-            public SqlGenericsConverter(IConfiguration configuration)
+            public SqlGenericsConverter(IConfiguration configuration, ILoggerFactory loggerFactory)
             {
                 this._configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+                this._logger = loggerFactory?.CreateLogger(LogCategories.Bindings) ?? throw new ArgumentNullException(nameof(loggerFactory));
```

Comment:
Instead of passing in the logger factory and creating a new logger for each converter I'd suggest we instead create the logger in the config provider and pass that in directly.